lib/pacemaker/pcmk_injections.c

```diff
@@ -602,39 +619,35 @@ pcmk__inject_scheduler_input(pe_working_set_t *data_set, cib_t *cib,
 
         out->message(out, "inject-modify-ticket", "Granting", ticket_id);
 
-        rc = set_ticket_state_attr(out, ticket_id, "granted", "true",
-                                   cib, cib_sync_call | cib_scope_local);
-        CRM_ASSERT(rc == pcmk_ok);
+        rc = set_ticket_state_attr(out, ticket_id, "granted", "true", cib);
```

Comment:
It looks like every caller of set_ticket_state_attr is passing "true" or "false", so it could be changed to just take a bool.

Reply:
Good idea, added a commit on top